whist/core/game/trick.py

```diff
@@ -59,8 +59,11 @@ def play_card(self, player: PlayerAtTable, card: Card) -> None:
             raise NotPlayersTurnError(player.player, self.play_order[turn].player)
         if not LegalChecker.check_legal(player.hand, card, self.stack.first):
             raise ServSuitFirstWarning()
+        if card not in player.hand:
+            raise CardNotInHandError(f'{card} is not in {player}\'s hand')
 
         self.stack.add(card)
+        player.hand.remove(card)
```

Comment:
Isn't the card removed somewhere else in the code?

Reply:
I don't think so, but would have to check.

Reply:
I think previously we popped it directly in the test.